src/layer/x86/convolution_3x3_pack8.h

```diff
@@ -76,7 +66,7 @@ static void conv3x3s1_winograd64_transform_kernel_pack8_avx(const Mat& kernel, M
     // interleave
     // src = 64-inch-outch
     // dst = 4b-4a-inch/4a-64-outch/4b;
-    kernel_tm_pack8.create(2 * inch / 8, 64, (outch / 4) / 2 + (outch / 4) % 2, (size_t)4u * 64, 64);
+    kernel_tm_pack8.create(inch / 8, 64, (outch / 4) / 2 + (outch / 4) % 2, (size_t)4u * 64, 64);
```

Comment:
outch must be a multiple of 8 here
(outch / 4) / 2 + (outch / 4) % 2 to outch / 8

Reply:
see above